cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/TwoWindingsTransformerConversion.java

```diff
@@ -134,8 +163,34 @@ private void addTapChangers(TwoWindingsTransformer tx) {
                     ptc);
             invalid(reason);
         }
+
+        setRegulatingControlContext(tx, rtc, ptc);
+    }
+
+    private void setRegulatingControlContext(TwoWindingsTransformer tx, String rtcId, String ptcId) {
+        RegulatingControlRatio rcRtc = null;
+        if (rtcId != null) {
+            PropertyBag tc = powerTransformerRatioTapChanger.get(rtcId);
+            rcRtc = context.regulatingControlMapping().forTransformers().buildRegulatingControlRatio(rtcId, tc);
+        } else {
+            rcRtc = context.regulatingControlMapping().forTransformers().buildEmptyRegulatingControlRatio();
```

Comment:
Is it really useful? I feel like this should just be a null `rcRtc` rather than building a whole new empty object.

Reply:
Done